### ISSUE TITLE
Update Admob installation documentation

### ISSUE THREAD

**Issue by saiichihashimoto** [googleads/googleads-mobile-unity#555]
I installed the plugin (Version 3.8.0) and attempted to use it, getting these XCode Errors:

```
Unexpected '@' in program
Property 'safeAreaLayoutGuide' not found on object of type 'UIView *'
```

![image](https://user-images.githubusercontent.com/2819256/32194278-28504376-bd77-11e7-8f5c-2a56b9a81179.png)

I can't imagine I did anything wrong, I just plugged it in.

**Comment by exodo24** [comment]
I also have the same issue ! Have you been able to resolve?

**Comment by saiichihashimoto** [comment]
I haven't been able to, no. Maybe @rampara or @ericleich can address this?

**Comment by hass101** [comment]
Hi, I am also getting this error. thanks

**Comment by exodo24** [comment]
I updated to Xcode 9.0 and fixed this issue for me .

**Comment by saiichihashimoto** [comment]
Somehow, my comment didn't post. I did the same and it fixed it for me. However, this issue isn't really solved since [the website for installing this](https://developers.google.com/admob/unity/start) claims:

![image](https://user-images.githubusercontent.com/2819256/32429896-97e89e4a-c281-11e7-94e9-9856bc81f712.png)

This needs to get fixed.

**Comment by stowy** [comment]
Thanks for bringing this to our attention. It will be fixed to be compatible with lower versions of Xcode in the next release.

**Comment by rampara** [comment]
This has been fixed in the latest version of the plugin (3.9.0).
